MedbayTech.Appointment/Domain/Entities/AppointmentRealocation.cs

```diff
@@ -10,13 +11,22 @@ public class AppointmentRealocation : IIdentifiable<int>
         [Key]
         [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
         public int Id { get; set; }
+        public AppointmentRealocationType AppointmentRealocationType { get; set; }
         public DateTime Start { get; set; }
         public DateTime End { get; set; }
         public bool Deleted { get; set; }
-        public bool Finished { get; set; }       
+        public bool Finished { get; set; }
+        public bool IsCanceled { get; set; }
+        public string Description { get; set; }
         public int RoomId { get; set; }
         [NotMapped]
         public virtual Room Room { get; set; }
+        public RoomType RoomOneType { get; set; }
+        public string RoomOneUse { get; set; }
+        public string RoomOneLabel { get; set; }
+        public RoomType RoomTwoType { get; set; }
+        public string RoomTwoUse { get; set; }
+        public string RoomTwoLabel { get; set; }
         public int HospitalEquipmentId { get; set; }
         [NotMapped]
         public virtual HospitalEquipment HospitalEquipment { get; set; }
```

Comment:
I don't like the way appointment realocation is configured. You could make an abstract class with the properties all realoactions have, and then you could make types of realocations that inherit that one class. You will still have one table, it will only be more readable.